### PR TITLE
Cleanup `manifest.json`

### PR DESCRIPTION
**Pull request by fregante**
Tiny changes

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -10,8 +10,7 @@
     "128": "assets/icons/favicon-128.png"
   },
   "description": "Lightning up the browser",
-  "homepage_url": "https://github.com/bumi",
-  "short_name": "Lightning",
```
Comment (fregante) [inline src/manifest.json:14]: Duplicate of the regular name

File: src/manifest.json [hunk 2]
```diff
@@ -20,8 +19,7 @@
     "storage",
     "webRequest",
     "webRequestBlocking",
-    "http://*/*",
-    "https://*/*"
```
Comment (fregante) [inline src/manifest.json:24]: Exactly equivalent to *://*/*

File: src/manifest.json [hunk 3]
```diff
@@ -31,7 +29,7 @@
     "name": "bumi"
   },
 
-  "__firefox__applications": {
+  "__firefox__browser_specific_settings": {
```
Comment (fregante) [inline src/manifest.json:32]: It was renamed: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings

File: src/manifest.json [hunk 4]
```diff
@@ -53,11 +51,11 @@
     "__firefox__browser_style": false
   },
 
-  "__chrome|opera__options_page": "options.html",
   "options_ui": {
     "page": "options.html",
     "open_in_tab": true,
-    "__chrome__chrome_style": false
+    "__chrome|opera__chrome_style": false,
+    "__firefox__browser_style": false
```
Comment (fregante) [inline src/manifest.json:58]: They all have it. Was this intentional? https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/options_ui

Comment (bumi) [reply src/manifest.json:58]: no, was not intentional. thanks!

**Review by bumi — APPROVED** [review]
super! thanks for the review and cleanup!

**Pull request merged**